models/base/scratch/snowplow_unified_base_events_this_run.sql

```diff
@@ -37,8 +37,13 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
     {% do contexts.append({'schema': var('snowplow__cwv_events'), 'prefix': 'cwv_', 'single_entity': True}) %}
   {% endif -%}
 
-  {% if var('snowplow__enable_browser_context', false) -%}
+  {% if var('snowplow__enable_browser_context', false) and not var('snowplow__enable_browser_context_2', false) %}
```

Comment:
Decided to not remove 1s in case both are enabled to do the coalesce below (not to have to test things again locally) in which case it has to be in one if else block to separate.